documentation/docs/steps/apiKeyValueMapUpload.md

```diff
@@ -0,0 +1,44 @@
+# ${docGenStepName}
+
+## ${docGenDescription}
+
+With this step you can store one or more key value pairs of data stored in a group called a map or key value map.
+
+To consume the ApiKeyValueMapUpload step, proceed as follows:
+
+* Copy the SAP API management service key from the SAP BTP sub account cockpit, under instance and subscriptions-->service API Management,API portal, which was created under apiportal-apiaccess plan.
```

Comment:
```suggestion
* Copy the SAP API management service key from the SAP BTP sub account cockpit, under instance and subscriptions-->service API Management, API portal, which was created under apiportal-apiaccess plan.
```

Reply:
I'm not sure the `-->` will look good in the rendered document. We should build it again and see.